cla_backend/settings/base.py

```diff
@@ -341,6 +342,27 @@ def traces_sampler(sampling_context):
 SECURE_CONTENT_TYPE_NOSNIFF = True
 SESSION_COOKIE_SAMESITE = "strict"
 
+CSP_DEFAULT_SRC = [
+    "'self'",
+    "o345774.ingest.sentry.io",
+    "www.googletagmanager.com",
```

Comment:
Not sure if we need google and doubleclick stuff since we're not using Google analytics and tagman on backend. Or are we?